docs/feature-variables-and-me.md

```diff
@@ -0,0 +1,266 @@
+
+> 👋 Unimplemented
+>
+> *Coming soon* While this is available in Experimenter, currently the Nimbus SDK does not support different features in different branches.
+
+We might also imagine a world where we have multiple features as before. Two differnet product teams are experimenting with two new capabilities of the app: both require onboarding instructions, one has an entry point via a app menu item, and the other has an entry point in the new tab screen.
+
+If it was one product team where communication is high, perhaps they might run one experiment, with two treatment branches: one branch with configuration for the `onboarding` and `app-menu` features, and one branch with configuration for the `onboarding` and `new-tab` features.
+
+Both teams require the `onboarding` feature. This allows each team to run their own experiments, which do not interfere with one another.
+
+> ⛅️🔬🔭  Concept
+>
+> While for any given user a feature may be involved in only one experiment, one experiment should be able to configure multiple features.
+
+Because both product teams' experiments require the `onboarding` experiment, no user will be involved in _both_ experiments.
+
+> 👋 Unimplemented
+>
+> *Coming soon* Configuring multiple features per branch isn't yet in Experimenter.
+
+## Working with configurable features
+
+Throughout the process of designing and building these configurable features, the feature variables have needed to be documented. At first, when the feature is being envisioned and designed, the variables should travel with the designs themselves.
+
+When the feature is being implemented, these variables will begin to acquire concrete names, types and organization, which will be used extract JSON from the Nimbus SDK and configure the application features themselves. This documentation will begin to take shape and textual organization that travel in the app's repository.
+
+When the feature is being tested, QA testers are going to want to configure the features within bounds and tolerances set by the designs and the engineers.
+
+Finally, when the feature is part of experiments, then the experiment owner, setting the branches in Experimenter needs to be able configure the branches with variables with spellings and organization that match the app.
+
+It's strongly recommended that you keep a track of these identifiers and descriptions of the configurability of each of the features in at least one place.
+
+> 👋 Unimplemented
+>
+> The Nimbus team has a few ideas about this, for documenting the features in an app in a machine readable format, tentantively called the Manifest. We're thinking:
+> 
+> 1. generating code for app developers to use in configuration.
+> 2. generating UI for experimenters to use in experiment design
+> 3. generating UI for QA to use in feature testing and local development.
+>
+> We're in the early stages of planning this, so would love to hear some feedback or ideas you may have.
```

Comment:
This would be extremely helpful. We are experimenting with this currently, so let's talk more.